cypress/integration/basic/role_assignment_deleted.spec.js

```diff
@@ -0,0 +1,73 @@
+const BOOKMARKED_PERSON = 'Sadaf Arshad';
+
+beforeEach(() => {
+  Cypress.Cookies.preserveOnce('_DSS-RM_session');
+});
+
+describe('Test that role assignments can be made', () => {
```

Comment:
Update describe() and it() language as described here: https://github.com/dssit/roles-management/pull/349